zeppelin-server/src/test/java/org/apache/zeppelin/AbstractZeppelinIT.java

```diff
@@ -147,10 +129,15 @@ protected void createNewNote() {
   protected void deleteTestNotebook(final WebDriver driver) {
     driver.findElement(By.xpath(".//*[@id='main']//button[@ng-click='removeNote(note.id)']"))
         .sendKeys(Keys.ENTER);
-    sleep(1000, true);
+    ZeppelinITUtils.sleep(1000, true);
     driver.findElement(By.xpath("//div[@class='modal-dialog'][contains(.,'delete this notebook')]" +
         "//div[@class='modal-footer']//button[contains(.,'OK')]")).click();
-    sleep(100, true);
+    ZeppelinITUtils.sleep(100, true);
+  }
+
+  protected void clickAndWait(final By locator) {
+    driver.findElement(locator).click();
+    ZeppelinITUtils.sleep(1000, true);
```

Comment:
Looks great! 
Are there any reason not to use polling facilities of [pollingWait](https://github.com/prabhjyotsingh/zeppelin/blob/master/zeppelin-server/src/test/java/org/apache/zeppelin/AbstractZeppelinIT.java#L108) ?